platforms/core-configuration/kotlin-dsl-integ-tests/src/integTest/kotlin/org/gradle/kotlin/dsl/integration/K2IntegrationTest.kt

```diff
@@ -101,7 +101,6 @@ class K2IntegrationTest : AbstractKotlinIntegrationTest() {
 
     private
     fun withK2BuildLogic() {
-        withFile("build-logic/gradle.properties", "kotlin.experimental.tryK2=true")
```

Comment:
❌ We must keep this otherwise we don't test K2 here

Reply:
Isn't the property deprecated? What is the new way to run with K2?

Reply:
You are basically saying that this change on `master` is wrong: https://github.com/gradle/gradle/pull/28744/files#diff-80d478a77cc124b7ee2998660cf46777d66e0fdafb6c6821ad9958a6affe08b0L104

@ov7a What is your take on that?

Reply:
@eskatos is correct

Reply:
After discussing it a bit, we figured out that the original change is correct, so the changes are correct and nothing to be done.